README.md

```diff
@@ -2,7 +2,7 @@
 # FAST-DNA overview
 FAST-DNA is tooling, resources, and components used to accelerate building Web sites and applications.
 
-Teams spend significant time building parts of the design system (colors, variable font, typography, grid, components, etc.) which can slow adoption of the latest design guidelines and related technologies. FAST-DNA solves this problem by providing tooling and resources to accelerate development with code that is performant and accessible. The resources work with multiple component frameworks such as (Fabric, Foundation, etc.) as well as JavaScript frameworks and libraries (React, Angular, etc.).
+Teams spend significant time building parts of the design system (colors, variable font, typography, grid, components, etc.) which can slow adoption of the latest design guidelines and related technologies. FAST-DNA solves this problem by providing tooling and resources to accelerate development with code that is performant and accessible. The resources work with multiple component frameworks such as (Fabric, Foundation, etc.) as well as JavaScript frameworks and libraries (React, etc.).
```

Comment:
I think that whole sentence could be adjusted to paint a clearer picture. I think the following sentence is more illustrative of the current project:

"In addition to a number of React packages, many of these resources are written to be framework agnostic and will work with any component library (Fabric, Foundation, etc.) and JavaScript framework (React, Angular, etc.)."